docs/readmes/howtos/ue_ip_address_management.md

```diff
@@ -1,16 +1,16 @@
 ---
 id: config_agw_bridged
-title: AGW Bridged Mode
+title: AGW Non-NAT Mode
 hide_title: true
 ---
-# Magma AGW Bridged mode
+# Magma AGW Non-NAT mode
 
-Bridged mode on the Access Gateway refers to any IP allocation strategy that
+Non-NAT mode on the Access Gateway refers to any IP allocation strategy that
 does not NAT assigned UE IP addresses at the AGW.
 
 There are 3 components to reconfigure to enable bridged mode:
 
-- Bridged (Non-NAT) config option for AGW
+- Bridged/Routed (Non-NAT) config option for AGW
```

Comment:
Just checking - everywhere else in this PR you moved `Bridged` to `Non-NAT`, but here you moved `Bridged (Non-NAT)` to `Bridged/Routed (Non-NAT)` - just making sure this is clear to the user? The change appears inconsistent to me but may be fine!

Reply:
ok, let me fix it.